subprojects/packagefiles/taocpp-json/meson.build

```diff
@@ -0,0 +1,16 @@
+project(
+    'taocpp-json',
+    'cpp',
+    meson_version: '>=0.60.0',
+    version: '1.0.0-beta.14',
+    license: 'MIT',
+    default_options: [
+        'cpp_std=c++17',
+    ]
+)
+
+taocpp_json_inc = include_directories('include')
+
+taocpp_json_dep = declare_dependency(
+    include_directories: [taocpp_json_inc]
+)
```

Comment:
The upstream CMakeLists.txt depends on PEGTL, why doesn't this?

Reply:
taocpp-json is header-only, PEGTL is only required for tests.

Reply:
What does "header-only" have to do with it? How is someone expected to utilize taocpp-json if PEGTL is not installed and checked for?

Reply:
not my day... all my assumptions are based on the Debian's package which has not been updated for 4 years, and at that time, taoJSON embedded PEGTL.